pkg/ocp4/ocp4.go

```diff
@@ -50,18 +60,47 @@ func (ocp4Node *Node) Translate(ocp3Node configv1.NodeConfig) {
 // GenYAML returns the list of translated CRDs
 func (ocp4Master *Master) GenYAML() ([]Manifest, error) {
 	var manifests []Manifest
-	if ocp4Master.OAuth.Kind != "" {
-		manifest := Manifest{Name: "100_CPMA-cluster-config-oauth.yaml", CRD: ocp4Master.OAuth.GenYAML()}
+
+	// Generate yaml for oauth config
+	crd := ocp4Master.OAuth.GenYAML()
+	manifests = appendOauthManifests(ocp4Master.OAuth.Kind, crd, manifests)
+
+	// Generate yaml for oauth secrets
+	manifests = appendOauthSecrets(ocp4Master.Secrets, manifests)
+
+	// Generate yaml for SDN config
+	networkCR := ocp4Master.Network.GenYAML()
+	manifests = appendSDNManifest(networkCR, manifests)
+
+	return manifests, nil
+}
+
+func appendOauthManifests(oauthCRKind string, crd []byte, manifests []Manifest) []Manifest {
+	if oauthCRKind != "" {
+		manifest := Manifest{Name: "100_CPMA-cluster-config-oauth.yaml", CRD: crd}
```

Comment:
I know you didn't introduce it but it could be a good opportunity to use a constant for the prefix part of the Name: `100_CPMA-cluster...`